packages/esm-service-queues-app/src/queue-table/cells/queue-table-extension-cell.component.tsx

```diff
@@ -1,23 +1,10 @@
 import React from 'react';
-import { useRenderableExtensions } from '@openmrs/esm-framework';
+import { ExtensionSlot } from '@openmrs/esm-framework';
 import { type QueueTableColumnFunction, type QueueTableCellComponentProps } from '../../types';
-import { type ExtensionColumnConfig } from '../../config-schema';
-
-export const queueTableExtensionColumn: QueueTableColumnFunction = (key, header, config: ExtensionColumnConfig) => {
-  const state = config?.state ?? {};
 
+export const queueTableExtensionColumn: QueueTableColumnFunction = (key, header) => {
   const QueueTableExtensionCell = ({ queueEntry }: QueueTableCellComponentProps) => {
-    const extensions = useRenderableExtensions('queue-table-extension-column-slot');
```

Comment:
Yeah `useRenderableExtensions` AFAICT has all the same overhead as an extension slot. Used the way as it is at the cell level there's no real difference: compare with the [ExtensionSlot](https://github.com/openmrs/openmrs-esm-core/blob/eeb48a4c573111d7323956164d75e7f64723543b/packages/framework/esm-react-utils/src/ExtensionSlot.tsx) implementation. I don't know how this hook could be used to make things lighter. Maybe @ibacher has an idea. I think it's unlikely that this kind of thing will work until we have [framework-native mounting](https://openmrs.atlassian.net/browse/O3-1418).

Reply:
Thanks @chibongho for the heads up. CC @donaldkibet @makombe

Reply:
Hmmm... the idea here was basically to end up with a parcel-per-column and then feed each parcel the row data based on props, e.g., so that we end up with 8 parcels rendered # rows times rather than 8 * # rows parcels. That said, it may not be a substantial reduction in rendering times. I think implementation-wise, that'd only work if `const extensions` is assigned outside of the `QueueTableExtensionCell` component though.